src/components/current-query/_fs-applied-filters.scss

```diff
@@ -5,12 +5,12 @@
  * @copyright Copyright (c) 2017-2019 Palantir.net
  */
 
-.fs-applied-filters {
+#fs-root .fs-applied-filters {
```

Comment:
@biz123 If I don't add the ID string here, the filters get clobbered by global style rules from `#fs-root button, #fs-root input, #fs-root optgroup, #fs-root select, #fs-root textarea`

Should we be namespacing all of our CSS like this?